README.rst

```diff
@@ -47,11 +47,13 @@ This plugin adds the following options to the csv-table_ directive:
     This option takes a Python dict specifying the column index (starting at
     zero) and a regular expression. Rows are included if the columnar value
     matches the supplied regular expression.
+    If multiple filters are given, row is included if all columns match.
 
 ``:exclude:``
     This option takes a Python dict specifying the column index (starting at
     zero) and a regular expression. Rows are excluded if the columnar value
     matches the supplied regular expression.
+    If multiple filters are given, row is excluded if any columns match.
```

Comment:
This is not the case, if I'm not mistaken. We exclude when _any_ exclude filter matches, not when _all_ exclude filter match. And I'd like to keep it that way.

To make the difference to `:include:` clear, let's rephrase the last sentence:

`Rows are excluded if the columnar value matches *any* supplied regular expression.`